### PR TITLE
Add new attributes to EburyAssociation, add EburyAssociationAccount

### PR DESCRIPTION
**Pull request by english**

### CODE REVIEW

File: swagger.yaml [hunk 1]
```diff
@@ -4599,6 +4599,163 @@ paths:
...
+        400:
+          $ref: "#/responses/400BadRequest"
+        401:
+          $ref: "#/responses/401Unauthorized"
+        403:
+          $ref: "#/responses/403Forbidden"
+        404:
+          $ref: "#/responses/404NotFound"
+        409:
+          $ref: "#/responses/409Conflict"
```
Comment (aleksandra-form3) [inline swagger.yaml:4751]: Similarly, is it possible to get a 409 for a deletion?

File: swagger.yaml [hunk 2]
```diff
@@ -4599,6 +4599,163 @@ paths:
...
+            $ref: "#/definitions/EburyAssociationAccountResponse"
+        400:
+          $ref: "#/responses/400BadRequest"
+        401:
+          $ref: "#/responses/401Unauthorized"
+        403:
+          $ref: "#/responses/403Forbidden"
+        404:
+          $ref: "#/responses/404NotFound"
+        409:
```
Comment (aleksandra-form3) [inline swagger.yaml:4707]: Is it possible to get a 409 for fetching a resource?

File: swagger.yaml (hunk 1 above)
Comment (aleksandra-form3) [reply swagger.yaml:4751]: Scrap that, I saw the implementation in another PR.

File: swagger.yaml (hunk 2 above)
Comment (english) [reply swagger.yaml]: Good catch! This seems to have been copy-pasted all over the place. I've removed it from the GET endpoints for our association and association account in 12e09d28fe7b34d6673b035e3921b206179df410

Comment (aleksandra-form3) [reply swagger.yaml]: :+1:

**Review by aleksandra-form3 — APPROVED** [review]

**Pull request merged**